### PR TITLE
fix: Dark theme: Consumers: theme colors issues

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)
bug: Dark theme: Consumers: theme colors issues

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

closes: #3944 
**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings (e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/globalCss.ts [hunk 1]
```diff
@@ -110,5 +110,25 @@ export default createGlobalStyle(
...
       }
     }
+    .rmsc {
+      --rmsc-main: #4285f4;
+      --rmsc-hover: #f1f3f5;
+      --rmsc-selected: #e2e6ea;
+      --rmsc-border: #ccc;
+      --rmsc-gray: #aaa;
+      --rmsc-bg: #fff;
+      --rmsc-p: 10px; /* Spacing */
```
[the remark this answers is not in the document]
Comment (Haarolean) [reply kafka-ui-react-app/src/components/globalCss.ts:120]: guess not?

[the remark this answers is not in the document]
Comment (tarunsamanta2k20) [reply kafka-ui-react-app/src/components/globalCss.ts:120]: first confirm if it's working or not then fix the styling.

[the remark this answers is not in the document]
Comment (Haarolean) [reply kafka-ui-react-app/src/components/globalCss.ts:120]: styling is the core of the issue, so what do we consider as "working"?
image
Current colors are too dark, feel free to use the colors present at [our figma page](https://www.figma.com/file/ZkvysdRUmt9p2PDNU3a9WU/Kafka-UI?type=design&node-id=2040-5727)
Our "reset offsets" page has been [updated](https://www.figma.com/file/ZkvysdRUmt9p2PDNU3a9WU/Kafka-UI?type=design&node-id=2633-13933&t=gyNW2VvP2gTsay5g-0) there but not yet implemented, so there's no strict reference for now.

[the remark this answers is not in the document]
Comment (tarunsamanta2k20) [reply kafka-ui-react-app/src/components/globalCss.ts:120]: Is the existing color dark grey?

[the remark this answers is not in the document]
Comment (tarunsamanta2k20) [reply kafka-ui-react-app/src/components/globalCss.ts:120]: As I can't run your app. Not test once.

**Comment by tarunsamanta2k20** [comment]
if everything has been done for this PR, then please merge. if any changes are required then please let me know. Since I can't run your app.

**Comment by Haarolean** [comment]
image
still doesn't match the other components in the app.
Like this:
image

Please figure out how to run the app and the needed components.

An example config for a dummy connector/consumer:
```
{"name": "local-file-sink",
"connector.class": "FileStreamSink",
"tasks.max": "1",
"file": "/tmp/test.sink.txt",
"topics": "connect-test"
}
```

compose file modifications:
For kafka-connect0 replace `CONNECT_PLUGIN_PATH` with the following:
`CONNECT_PLUGIN_PATH: "/usr/share/java,/usr/share/confluent-hub-components,/usr/share/filestream-connectors"`

**Comment by Haarolean** [comment]
@tarunsamanta2k20 please provide some screenshots with the changes

**Comment by Haarolean** [comment]
Once again, to make this as clear as possible: the duty of providing evidence of the changes lies upon the contributor itself.
For this particular PR we'd need screenshots of the affected area in UI.

**Pull request closed without merging**